roles/cluster/cluster_deploy_ldap/tasks/main.yml

```diff
@@ -144,9 +144,12 @@
   # Workaround until `ocm` supports the --insecure flag
 
   - name: Get the cluster ID
-    shell:
+    shell: |
+      set -o pipefail
       ocm describe cluster "{{ cluster_deploy_ldap_cluster_name }}" --json | jq .id -r
     register: cluster_id_cmd
+    args:
+      executable: /bin/bash
```

Comment:
not sure why you add this `args.executable`  🤔 To force bash, vs having bash or sh?

Reply:
Yeah about this, in my case I use ZSH and Ansible is not able to determine what is the default interpreter so it breaks, if this is intended to run I can remove it everywhere to make the PR easier to review ++

Reply:
hum, strange, I've been using zsh as my main shell forever, and I never hard issues 🤔 
what about all the other `shell:` tasks?
can you show me how Ansible complains?